packages/ember-data/tests/integration/adapter/rest-adapter-test.js

```diff
@@ -33,10 +33,10 @@ module("integration/adapter/rest_adapter - REST Adapter", {
 });
 
 function ajaxResponse(value) {
-  adapter.ajax = function(url, verb, hash) {
-    passedUrl = url;
-    passedVerb = verb;
-    passedHash = hash;
+  adapter.ajax = function(request) {
+    passedUrl = request.url;
+    passedVerb = request.method;
+    passedHash = request.body ? { data: request.body } : undefined;
```

Comment:
I didn't want to update all assertions below; I think this can be refactored/simplified more once this is accepted ...